suite-native/module-onboarding/src/screens/AnalyticsConsentScreen.tsx

```diff
@@ -48,36 +111,89 @@ export const AnalyticsConsentScreen = () => {
     };
 
     return (
-        <OnboardingScreen title="User data consent" activeStep={3} isScrollable>
-            <VStack style={applyStyle(analyticsConsentStyle)}>
-                <Card>
+        <Screen isScrollable>
+            <Box
```

Comment:
nit: I'd personally create a component out of this box for better readability of this screen without having to read lot of JSX but I'll leave that up to your decision, personal preference

Reply:
I dont think it will help here much and it will add fregmentation of where the code is. I'll keep it this way this time.